prac_07/guitar.py

```diff
@@ -1,6 +1,6 @@
 """
 CP1404 Practical
-Classes for guitars.py program
+Classes for guitars.py program.
 """
 
 
```

Comment:
In your __str__ method
```
    def __str__(self):
        """Return string formatting. Example: Gibson L-5 CES (1922) : $16,035.40"""
        return f"{self.name}, {self.year}, {self.cost:,.2f}"
```

Your current example in the docstring doesn't accurately represent _**how**_ it is being returned (The actual attributes are correct) - Your return function separates the attributes by commas, whilst your docstring indicates otherwise.

I would recommend something like this to better match your docstring:
```
    def __str__(self):
        """Return string formatting. Example: Gibson L-5 CES (1922) : $16,035.40"""
        return f"{self.name} ({self.year}), : {self.cost:,.2f}"
```

Reply:
Fair point, I'll actually change the docstring because the return f"{self.name}, {self.year}, {self.cost:,.2f}" is the format I needed for the writing to the .txt file. Just forgot to update the docstring.